vendor/github.com/prometheus/alertmanager/notify/notify.go

```diff
@@ -284,6 +285,11 @@ func NewMetrics(r prometheus.Registerer, ff featurecontrol.Flagger) *Metrics {
 			Name:      "notification_requests_failed_total",
 			Help:      "The total number of failed notification requests.",
 		}, labels),
+		numNotificationSuppressedTotal: prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
@gotjosh I think I need to add this metric [here](https://github.com/grafana/mimir/blob/31a14b5e6ddceafcf6277089aae184db61637905/pkg/alertmanager/alertmanager_metrics.go#L107)?